0000-server-roles.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Interledger node roles
+draft: 1
+---
+# Interledger node roles
+
+This document describes the different roles a node can play on the Interledger main net or testnet: sender, connector, or receiver.
+
+## Bilateral links
+
+All network participants should connect to one or more other network participants using one of four bilateral transfer protocols:
+BTP/2.0-I, HTTP-OER-ASYNC, or HTTP-OER-SYNC. Note that for the two BTP/2.0 variations, one neighbor plays the server role; the other plays the client role.
+For the two HTTP-OER variations, both neighbors implement an equal role.
+
+Depending on which settlement ledger is used, side protocols (i.e., protocols with a protocolName other than `'ilp'`) should be implemented accordingly. For instance, https://github.com/interledgerjs/ilp-plugin-xrp-asym-server/issues/10 describes a number of additional protocols to be used for sending XRP payment channel claims.
```

Comment:
Rather than reference a transient issue I would reference the plugin that uses side-protocols and give an example of one.